packages/adapter-vercel/index.d.ts

```diff
@@ -6,9 +6,9 @@ export default function plugin(config?: Config): Adapter;
 export interface ServerlessConfig {
 	/**
 	 * Whether to use [Edge Functions](https://vercel.com/docs/concepts/functions/edge-functions) (`'edge'`) or [Serverless Functions](https://vercel.com/docs/concepts/functions/serverless-functions) (`'nodejs18.x'`, `'nodejs20.x'` etc).
-	 * @default 'nodejs18.x'
+	 * @default Same as the build environment
 	 */
-	runtime?: `nodejs${number}.x`;
+	runtime?: 'nodejs18.x' | 'nodejs20.x';
```

Comment:
I'd like to keep this as the type because Vercel may deprecate old versions after some time and/or introduce new ones, and we don't want people to upgrade to newer versions of the adapter just for silencing type errors / don't want to make a major version bump just because we gotta remove a type
```suggestion
	runtime?: `nodejs${number}.x`;
```